docs/examples/good_practices/index.rst

```diff
@@ -1,5 +1,13 @@
-**************
-Good Practices
-**************
+*******************
+Good practices
+*******************
```

Comment:
```suggestion
==============
Good Practices
==============
```

btw, I think it would be ok to have only checkpointing alone so when we click on it in the left pane we land on the example directly

Reply:
We're going to add more examples in the "good practices" section soon. It's fine IMO if the section only has one page for now.

Reply:
Fixed in 65d6157897f45be881f5ce33dc1e99eb5bf817fc